contentful/apps
# feat: trigger hosted app action to send test notification [INTEG-1655]

## Purpose

This PR creates a hosted app action that calls the MS Teams Bot Service to send a test notification to a configured channel. It also updates the frontend to call the HAA when you click on the "Test" button for a notification.

## Approach

Followed some of the response/error handling patterns from other apps, and passing in environment variables in the same way that the Merge App does. There can be additional follow up work when we add other HAAs to improve some error handling.

## Testing steps

1. Follow the instructions in the README to set development environment variables
2. Run `npm run build` at the top level
3. Follow the instructions in the README to deploy the build to the sandbox app

Note: at this point the notification will not actually be sent, you should see an error "Team not found". This is because we are still using sample data for the channels instead of pulling in actual installations. You can actually send a message by manually changing the channel and team id to something where your development bot is installed.

## Breaking Changes

None

## Dependencies and/or References

## Deployment
